apps/src/code-studio/pd/workshop_dashboard/components/survey_results_header.jsx

```diff
@@ -72,35 +73,48 @@ const SurveyResultsHeader = React.createClass({
       return workshop.course === course;
     });
 
-    let firstWorkshopId = filteredWorkshops[0] ? filteredWorkshops[0].id : undefined;
+    let firstWorkshopId = this.props.organizerView ? undefined : (filteredWorkshops[0] ? filteredWorkshops[0].id : undefined);
 
     this.setState({
       selectedCourse: course,
       filteredWorkshops: filteredWorkshops,
       selectedWorkshopId: firstWorkshopId,
     });
 
-    this.setSurveyPanel(firstWorkshopId, this.getWorkshopFriendlyName(filteredWorkshops[0]));
+    this.setSurveyPanel(course, firstWorkshopId, firstWorkshopId && this.getWorkshopFriendlyName(filteredWorkshops[0]));
   },
 
-  setSurveyPanel(workshopId, workshopName) {
-    if (workshopId === undefined) {
+  setSurveyPanel(course, workshopId, workshopName) {
+    if (!this.props.organizerView && workshopId === undefined) {
```

Comment:
is `null` an acceptable value for workshopId here, as distinct from `undefined`? Otherwise be consistent: `if (!this.props.organizerView && !workshopId)`